packages/wdio-selenium-standalone-service/src/utils.ts

```diff
@@ -19,3 +31,17 @@ export function getFilePath (filePath: string, defaultFilename: string): string
 
     return absolutePath
 }
+
+/**
+ * find whether a browser session could be supported by the selenium-standalone service
+ * @param   {Capabilities.Capabilities} capabilities capabilities used for the session
+ * @returns {Boolean}                                true, if capabilities suggest a supported platform
+ */
+export function hasCapsWithSupportedBrowser (capabilities: Capabilities.Capabilities) {
+    if (!capabilities.browserName) {
+        return false
+    }
+    return SUPPORTED_CAPABILITIES.includes(
+        capabilities.browserName?.toLocaleLowerCase()
```

Comment:
```suggestion
        capabilities.browserName?.toLowerCase()
```

Reply:
The resason for this is that toLocaleLowerCase is not meant to be used like this.

Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLocaleLowerCase